package netty;

option java_package = "com.ease.nogame.protobuf";
option java_outer_classname = "PBMessage";

message MsgDesc {
    required string msgName = 1;
    optional bytes msgBytes = 2;
    optional int64 userId = 3;
    optional string token = 4;
    optional int32 errorCode = 5;
    optional string errorDesc = 6;
}

message PBHero {
        required int64 heroId = 1;
        required int64 dictId = 2;
        required int32 level = 3;
        required int32 exp = 4;
        repeated PBEquip equipList = 5;
}

message PBUser {
        required int64 userId = 1;
        required string userName = 2;
        required int32 level = 3;
        required int32 exp = 4;
        required int32 gem = 5;
        required int32 gold = 6;
}

message PBEquip {
        required int64 equipId = 1;
        required int64 dictId = 2;
        required int64 heroId = 3;
}

